components/global/product_switcher.tsx

```diff
@@ -70,6 +70,7 @@ const MenuItemTextContainer = styled.div`
     line-height: 20px;
 `;
 
+// TODO: extract this to a separate file
```

Comment:
Can we add tickets to all of these TODO comments?

Reply:
yep, will do. Thanks for pointing that out. 👍🏼